### PR TITLE
@damassi => Favorite artwork button turns into a bell during auction

### PR DESCRIPTION
**Pull request by erikdstock**
Resolves https://github.com/artsy/auctions/issues/525

This PR turns the Artwork save button into a bell if the artwork is in a not-closed auction. After the auction is closed, its state should remain the same but it will turn back into a heart.

screen shot 2017-07-26 at 3 42 21 pm
screen shot 2017-07-26 at 3 42 28 pm
screen shot 2017-07-26 at 3 42 34 pm
screen shot 2017-07-26 at 3 42 39 pm
screen shot 2017-07-26 at 3 42 53 pm
screen shot 2017-07-26 at 3 43 03 pm
screen shot 2017-07-26 at 3 43 10 pm
screen shot 2017-07-26 at 3 43 18 pm
screen shot 2017-07-26 at 3 43 23 pm
image

### CODE REVIEW

File: desktop/apps/artwork/components/actions/index.jade [hunk 1]
```diff
@@ -1,8 +1,14 @@
 .artwork-actions
-  a.artwork-action.artwork-action--save.black-tooltip(
-    data-message='Save'
-    class='js-artwork-save'
-  ): i.icon-heart
+  if artwork.is_in_auction
```
Comment (craigspaeth) [inline desktop/apps/artwork/components/actions/index.jade]: Pretty straightforward 👍

File: desktop/apps/artwork/components/metadata/test/template.coffee [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  jade.compile(
+    fs.readFileSync(filename),
+    { filename: filename }
+  )
+
+renderArtwork = (artworkOptions = {}, sdOptions = {}) ->
+  artwork = _.clone inAuctionArtwork.artwork
+  _.extend artwork, artworkOptions
+  sd = _.extend { CLIENT: artwork: artwork }, sdOptions
+  console.log artwork
```
Comment (craigspaeth) [inline desktop/apps/artwork/components/metadata/test/template.coffee]: I know it's WIP, but just in case there's a stray log here

**Comment by craigspaeth** [comment]
So far so good! Let me know if you need help with the tests.

**Comment by katarinabatina** [comment]
@erikdstock the bell seems huge on mobile web, can I get you another asset or can you size it down to be the same size as the share icon?

**Comment by erikdstock** [comment]
@katarinabatina yeah, my sizing is pretty arbitrary so any comments you have, the more specific the better.

I also want to confirm that the bell should turn back into a heart when the auction ends.

@craigspaeth the only logic in these views is whether the artwork is in an open auction. (plus rendering the icons) - if you have specific thoughts on testing I'm open but it seemed like it would be pretty straightfoward

**Comment by damassi** [comment]
LGTM, will merge on 🍏

**Pull request merged**